opsdroid/database/mongo/__init__.py

```diff
@@ -13,6 +14,7 @@
     "database": str,
     "user": str,
     "password": str,
+    "collection": str,
```

Comment:
isn't this: `default_collection_name` or `default_table_name`? Or is that somewhere else?

Reply:
Good catch. The `default_table_name` was proposed in a postgres related PR, but whatever we choose we should be consistent.